org/teams.yaml

```diff
@@ -102,6 +103,7 @@ teams:
           - rcaballeromx
           - rootsongjc
           - SataQiu
+          - Xunzhuo
```

Comment:
I don't think this is needed. Because docs maintainers/chinese is a child of docs maintainers, it should inherit those permissions. 

https://docs.github.com/en/organizations/organizing-members-into-teams/about-teams#nested-teams

```
Child teams inherit the parent's access permissions, simplifying permissions management for large groups. Members of child teams also receive notifications when the parent team is @mentioned, simplifying communication with multiple groups of people.

For example, if your team structure is Employees > Engineering > Application Engineering > Identity, granting Engineering write access to a repository means Application Engineering and Identity also get that access. If you @mention the Identity Team or any team at the bottom of the organization hierarchy, they're the only ones who will receive a notification.
```

Reply:
@brian-avery Ok I have removed it now~ Thanks